lib/codelog/message.rb

```diff
@@ -32,6 +32,10 @@ def missing_config_file
           "Run the following command to create the missing file:\n\n" \
           'codelog setup'
         end
+
+        def could_not_parse_yaml(error)
+          "#{prefix}: #{error.problem.capitalize} in #{error.file}:#{error.line}"
```

Comment:
@LukeDDC 
Maybe it would also be good to supply a suggestion about how to fix this error.

Something like:

> Try wrapping the whole line in quotes

Reply:
@emilianoLeite not all parsing errors can be solved by wrapping the string on quotes, do you think it could be misleading for the user in this cases?

Reply:
@luisbevilacqua We could add a conditional statement to verify if the error was caused by the lack of quotes in the line, and then print the suggested message